src/search.cpp

```diff
@@ -1232,7 +1232,7 @@ namespace {
 
       // For PV nodes only, do a full PV search on the first move or after a fail
       // high (in the latter case search only if value < beta), otherwise let the
-      // parent node fail low with value <= alpha and try another move.
+      // parent node fail low with value <= alpha and tries another move.
```

Comment:
I don't think this is correct

Reply:
That's right, this one should remain "try"
I have also undone the "a UCI chess-playing engine" change back to "a UCI chess playing engine"
because otherwise, we would also need to apply this change in all the other files of Stockfish